oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/rdb/RDBDocumentNodeStoreBuilder.java

```diff
@@ -113,4 +115,27 @@ public MissingLastRevSeeker createMissingLastRevSeeker() {
             return super.createMissingLastRevSeeker();
         }
     }
+
+    @Override
```

Comment:
Would it make sense to remove these, andhave default implementations in the interface?